.github/workflows/build-publish.yml

```diff
@@ -12,6 +12,9 @@ jobs:
   checks:
     name: "Checks"
     runs-on: ubuntu-24.04
+    permissions:
+      id-token: write
```

Comment:
checks shouldn't need `id-token: write` right?

Reply:
Oof. Thanks. This was a remnant from when I had GATI in here.

b71a02540d